frontend/src/Components/Headers/Header.module.scss

```diff
@@ -0,0 +1,80 @@
+@import "~plaid-threads/scss/variables";
+
+.grid {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  margin-bottom: 2 * $unit;
+}
+
+.title {
+  margin-top: 9 * $unit;
+  margin-bottom: 0;
+  font-weight: 800;
+  height: 6 * $unit;
+}
+
+.subtitle {
+  margin-top: 0;
+  margin-bottom: 3 * $unit;
+}
+
+.introPar {
+  width: 100%;
+  font-size: 2 * $unit;
+  margin: 2 * $unit 0;
+}
+
+.linkButton {
+  margin-top: 3 * $unit;
+  width: 200px;
+}
+
+.itemText {
+  text-decoration: underline;
+}
```

Comment:
this makes it look like a hyperlink to me. Would be cool to make it link to the docs glossary in a new tab!

Reply:
done